manuscript/How-secure-is-portugal.md

```diff
@@ -58,6 +61,19 @@ The real danger is from criminals who use the internet and vulnerable code for f
 
 We need to make the Portuguese network hostile to this type of criminal activity. The police and hackers (i.e. civil society) have an important role to play in this campaign.
 
+### Attackers' ROI (return on investment)
+
+We must remember that it doesn't take a lot of money to compromise a country or a company. This is the cost of the zero day, this is the worst case scenario to give (add table from slide 26 of C-days presentation).
+
```

Comment:
Done, but check that I've done it properly!  I'm Windows at the moment, hoping to get a Mac in the New Year.